projects/ngx-sub-form/src/lib/ngx-sub-form.component.spec.ts

```diff
@@ -306,38 +306,59 @@ describe(`NgxSubFormComponent`, () => {
         done();
       }, 0);
     });
+
+    it(`should correctly emit the onChange value only once when form is patched`, (done: () => void) => {
```

Comment:
That test title should be a little more explicit because I thought it was wrong at first (but it's not!)
We said earlier that we should never trigger the `onChange` callback when the form is patched from the parent so in that case it should when the form is patched locally I guess?

CF https://github.com/cloudnc/ngx-sub-form/blob/7015a1f4fe27bd2334d8628799bf2b3482c306c5/projects/ngx-sub-form/src/lib/ngx-sub-form.component.spec.ts#L276

Reply:
Fixed (clarified wording)